src/pages/index.tsx

```diff
@@ -14,18 +22,29 @@ interface IndexPageProps {
     allMarkdownRemark: {
       nodes: BlogPostTeaser[];
     };
+    officeImages: {
+      nodes: OfficeImage[];
+    };
   };
   location: Location;
 }
 
 const IndexPage = (props: IndexPageProps) => {
   const jobPositions = props.data.allSyPersonioJob.nodes;
   const blogPosts = props.data.allMarkdownRemark.nodes;
+  const officeImages = props.data.officeImages.nodes.reduce((memo, image) => {
+    memo[image.relativePath] = image;
+    return memo;
+  }, {});
```

Comment:
I want to work with a map of images so I can access them in the actual landingpage with `officeImages["office/01.png"]` and such. We need that because we pick selectors and we don't just show every office image.